scripts/background.js

```diff
@@ -1,13 +1,17 @@
-chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
+chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
   const tabUrl = tab.url ?? tab.pendingUrl;
   if (
-    changeInfo.status === 'complete' &&
+    changeInfo.status === "complete" &&
     tabUrl &&
-    (tabUrl.includes('youtube.com/results'))
+    tabUrl.includes("youtube.com/results")
   ) {
-    chrome.scripting.insertCSS({
-      target: { tabId: tabId },
-      files: ['css/global.css'],
-    });
+    try {
```

Comment:
I added  a try catch block when making this code asynchronous, but it's possible that this is unnecessary for the chrome object.